plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator_test.go

```diff
@@ -618,6 +618,99 @@ metadata:
 `)
 }
 
+func TestHelmChartInflationGeneratorValuesOverride(t *testing.T) {
```

Comment:
Accourding to documents, Legal value of `valuesMerge` are  ‘merge’, ‘override’ and ‘replace’.
https://kubectl.docs.kubernetes.io/references/kustomize/builtins/#_helmchartinflationgenerator_

Could you add any test cases of `valuesMerge : replace`?

Reply:
I added the test. I didn't add add it before, because `valuesMerge: replace`  doesn't use the changed code. There are also some other issues with replace behavior (not related to the change). I'll describe them in a PR comment.

Reply:
Thanks!